tests/test_models/test_track_heads/test_mask2former_head.py

```diff
@@ -0,0 +1,293 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+from mmengine import Config
+
+from mmtrack.models.track_heads import Mask2FormerHead
+from mmtrack.structures import TrackDataSample
+from mmtrack.testing import demo_mm_inputs
+
+
+class TestMask2FormerHead(TestCase):
+
+    def test_mask2former_head_loss(self):
+        config = Config(
+            dict(
+                type='Mask2FormerHead',
+                in_channels=[256, 512, 1024,
+                             2048],  # pass to pixel_decoder inside
+                strides=[4, 8, 16, 32],
+                feat_channels=256,
+                out_channels=256,
+                num_classes=40,
+                num_queries=100,
+                num_frames=2,
+                num_transformer_feat_level=3,
+                pixel_decoder=dict(
+                    type='MSDeformAttnPixelDecoder',
+                    num_outs=3,
+                    norm_cfg=dict(type='GN', num_groups=32),
+                    act_cfg=dict(type='ReLU'),
+                    encoder=dict(
+                        type='DetrTransformerEncoder',
+                        num_layers=6,
+                        transformerlayers=dict(
+                            type='BaseTransformerLayer',
+                            attn_cfgs=dict(
```

Comment:
duplicated config in `test_mask2former_head_loss` and `test_mask2former_head_predict`.

build mask2former head in `setUpClass`:https://github.com/open-mmlab/mmtracking/blob/dev-1.x/tests/test_datasets/test_base_video_dataset.py#L19